doc/sphinx/docs/python/tutorials/cec2006_gaco_benchmark.rst

```diff
@@ -0,0 +1,143 @@
+.. _py_tutorial_cec2006_gaco_benchmark_comp:
+
+ACO Benchmark Using the CEC2006 Problems
```

Comment:
Again use extended ACO or gaco .... this algorithm is not what the scientific community would recognise as ACO